source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -57,14 +57,15 @@ const NODE_STOPPED_STATES = [
 export default class NetworkStatusStore extends Store {
   // Initialize store properties
   _startTime = Date.now();
-  _tlsConfig: ?TlsConfig = null;
   _networkStatus = NETWORK_STATUS.CONNECTING;
   _networkStatusPollingInterval: ?IntervalID = null;
 
   // Initialize store observables
 
   // Internal Node states
+  @observable _tlsConfig: ?TlsConfig = null;
```

Comment:
@DeeJayElly observables should not have `_` prefix.

Reply:
> @DeeJayElly observables should not have `_` prefix.

Done